roles/ovirt-infra/README.md

```diff
@@ -51,6 +51,10 @@ The role has dependencies on multiple other oVirt roles. The following list disp
 
  This role accepts a variable called `permissions`. 
 
+From [ovirt-external-providers]
+
+This role expects a list called `external_providers`, which contains information about the external provider. To see example of this list please check the role's documentation.
```

Comment:
Please change this to:
``
This role accepts a variable called `external_providers`. 
```

It's new documentation format.